code-challenges/301/value-vs-reference/challenges-02.test.js

```diff
@@ -46,7 +46,7 @@ console.log(a) prints { fullName: 'Octavia Estelle Butler', yearBorn: 1947 }
 ------------------------------------------------------------------------------------------------ */
 
 const addBirthYearProperty = (obj, year) => {
-  // Solution code here...
+  obj.yearBorn = 1947;
```

Comment:
-1: This is a literal value being assigned instead of the function parameter value being assigned.